Sources/PersonalMessage.php

```diff
@@ -3776,6 +3782,7 @@ function ManageRules()
 	if (isset($_GET['apply']))
 	{
 		checkSession('get');
+		spamProtection('pm');
```

Comment:
Only note here is for 2.0 we did this because it wasn't feasible to apply a better fix.  This will trigger a warning if they send a PM then go apply rules right after.
Do we want to use a more clearly defined message/trigger here?

Reply:
> Do we want to use a more clearly defined message/trigger here?

Sounds lovely. How about we merge this, and then in a subsequent PR you can replace this with something better?